database/mssql/scripts/versions/v_11_ddl.sql

```diff
@@ -354,7 +354,6 @@ CREATE TABLE [dbo].[ORBC_USER_HIST]  (
 	[USER_GUID]                     	char(32) NOT NULL,
 	[USERNAME]                      	nvarchar(50) NOT NULL,
 	[USER_DIRECTORY]                	varchar(10) NOT NULL,
-	[USER_STATUS_TYPE]              	varchar(10) NOT NULL,
```

Comment:
We will need to alter the table in a new ddl such as v16 
CC: @john-fletcher-aot